app/views/pages/my_profile.html.erb

```diff
@@ -6,8 +6,9 @@
   <div class="profile-page-avatar">
       <%= cl_image_tag @user.photo.key, class: "profile-avatar-img" %>
   </div>
-  <div>
-      <%= link_to "Log out", destroy_user_session_path, data: {turbo_method: :delete}, class: "btn btn-outline-light m-3" %>
+  <div class>
```

Comment:
extra class here but not assigning anything -- double checked on Feline's rails s and log out button looks good and is working, so I'm happy to ignore